asllib/doc/ASLmacros.tex

```diff
@@ -554,7 +554,7 @@
 \newcommand\SHL[0]{\hyperlink{ast-shl}{\texttt{SHL}}} % Shift left for integers
 \newcommand\SHR[0]{\hyperlink{ast-shr}{\texttt{SHR}}} % Shift right for integers
 
-\newcommand\UNKNOWN[0]{\hyperlink{ast-unknown}{\texttt{UNKNOWN}}}
+\newcommand\ARBITRARY[0]{\hyperlink{ast-arbitrary}{\texttt{ARBITRARY}}}
```

Comment:
Should this link rather be `ast-earbitrary`?

Reply:
Ah good spot! Will fix this as part of https://github.com/herd/herdtools7/pull/1099/